snapcraft/commands/lxd

```diff
@@ -24,10 +24,6 @@ export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}:${SNAP_CURRENT}/lib:${SNAP_CURRENT}
 export PATH="${PATH}:${SNAP_CURRENT}/bin"
 export LXD_DIR=${LXD_DIR:-"${SNAP_COMMON}/lxd/"}
 
-# Make sure we have a ZFS binary on the path
-export LD_LIBRARY_PATH="${SNAP_CURRENT}/zfs-0.7/lib/:${LD_LIBRARY_PATH}"
-export PATH="${SNAP_CURRENT}/zfs-0.7/bin:${PATH}"
-
```

Comment:
Why is it unneeded?

Reply:
We need a functional `zfs` in PATH when running `lxd init` for it to work properly.

Reply:
I am assuming that `daemon.start` already took care of adding the proper ZFS version path to those variables. I don't see why that `0.7` version would need special treatment?